tests/NHSD.BuyingCatalogue.Solutions.API.UnitTests/SolutionsControllerTests.cs

```diff
@@ -1,16 +1,76 @@
-using System;
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using FluentAssertions;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
 using NHSD.BuyingCatalogue.Solutions.API.Controllers;
+using NHSD.BuyingCatalogue.Solutions.API.ViewModels.Solution;
+using NHSD.BuyingCatalogue.Solutions.Contracts;
+using NHSD.BuyingCatalogue.Solutions.Contracts.Queries;
 using NUnit.Framework;
 
 namespace NHSD.BuyingCatalogue.Solutions.API.UnitTests
 {
     [TestFixture]
     public sealed class SolutionsControllerTests
     {
+        private Mock<IMediator> _mockMediator;
+
+        private SolutionsController _solutionsController;
+
+        private const string SolutionId = "Sln1";
+
+        [SetUp]
+        public void Setup()
+        {
+            _mockMediator = new Mock<IMediator>();
+            _solutionsController = new SolutionsController(_mockMediator.Object);
+        }
+
         [Test]
         public void NullMediatorShouldThrowNullException()
         {
             Assert.Throws<ArgumentNullException>(() => new SolutionsController(null));
         }
+
+        [Test]
+        public async Task GetAsync_Solution_ReturnsExpectedGetSolutionResult()
+        {
+            var solution = Mock.Of<ISolution>(s => s.Id == SolutionId 
+                                                   && s.Name == "Some solution name"
+                                                   && s.Summary == "Some solution summary");
```

Comment:
Just a suggestion, but I think this would read better formatted like this:

```c#
var solution = Mock.Of<ISolution>(
    s => s.Id == SolutionId 
    && s.Name == "Some solution name"
    && s.Summary == "Some solution summary");
```
What do you think?